addons/mail/models/mail_channel.py

```diff
@@ -374,7 +374,7 @@ def _notify_email_recipients(self, message, recipient_ids):
         # real mailing list: multiple recipients (hidden by X-Forge-To)
         if self.alias_domain and self.alias_name:
             return {
-                'email_to': ','.join(formataddr((partner.name, partner.email)) for partner in whitelist if partner.email),
+                'email_to': ','.join(partner.email_formatted for partner in whitelist if partner.email),
```

Comment:
Isn't this a remaining of when mail group was on channel?

I don't see why we would some kind of alias for channels now.